gslib/utils/shim_util.py

```diff
@@ -53,6 +53,48 @@ class USE_GCLOUD_STORAGE_VALUE(enum.Enum):
 PRECONDITIONS_HEADERS = frozenset(
     ['x-goog-if-generation-match', 'x-goog-if-metageneration-match'])
 
+# The format for _BOTO_CONFIG_MAP is as follows:
+# {
+#   'Boto section name': {
+#     'boto field name': 'correspnding env variable name in Cloud SDK'
+#   }
+# }
+_BOTO_CONFIG_MAP = {
+    'Credentials': {
+        'aws_access_key_id': 'AWS_ACCESS_KEY_ID',
```

Comment:
Have these large dicts been getting duplicated?

Reply:
Are you looking at the latest commit? They were duplicated earlier because of a merge error, but they were fixed in later commit

Reply:
It looks like there's another dict like this in test_shim_util.py

Reply:
Oh I see what you mean. No, the test one is the actual boto config setting that we pass in. That' why all the values have `value` suffix there, to indicate that these are actual values.